.github/workflows/cfgov-python-satellite-ci.yml

```diff
@@ -0,0 +1,53 @@
+name: Python CI workflow
+
+on: [pull_request, push]
```

Comment:
I know our recipe has this, but I don't think we want both `pull_request` and `push` here. It's running every check twice on this PR. Maybe `push` only if it's a push to `master`?